source/functionality/search.md

```diff
@@ -0,0 +1,35 @@
+# Enable search
+
+The search function is especially useful for multipage documentation.
+
+## Amend the tech-docs.yml file
+
+Add the following to the end of your project’s `.../config/tech-docs.yml` file, or set it to true if it is already there:
+
+```
+# Enable search
+enable_search: true
+```
+
+Add the following to your `REPO/Gemfile`:
+
+```
+gem 'govuk_tech_docs'
+gem 'middleman-search-gds'
+```
+
+For example:
```

Comment:
Yes, that's a good point 🤔 
Also, let's avoid having an example of a 'good' Gemfile in the docs 😁 
If we stick to having the ready-to-go examples, they would have a Gemfile there. We could link to this Gemfile and there will be no need to maintain an extra copy. 👌 
Plus, if we have a list of docs websites using the TDT, there should be plenty of Gemfile examples there, too.

Reply:
I agree on all points @bravegrape and I think I misspoke. I don't think we should actually put a 'good' Gemfile anywhere because there are likely to be changes and so on, but I do think we need to point to good examples somewhere, which we obviously agree on.